html/pfappserver/lib/pfappserver/Model/Node/Tab/WMI.pm

```diff
@@ -108,7 +108,7 @@ sub parseWmi {
     my $scan_result = $scan->runWmi($scan_config, $rule_config);
     if ($scan_result =~ /ACCESS_DENIED/) {
         $rule_config->{item_exist} = "Access denied";
-    }elsif ($scan_result =~ /$WMI_NS_ERR/ || !@$scan_result) {
+    }elsif ($scan_result =~ /$WMI_NS_ERR/ || !$scan_result) {
```

Comment:
When error occurs the $scan_result is of type string which yields this error:
Can't use string ("NTSTATUS: NT_STATUS_HOST_UNREACH"...) as an ARRAY ref while "strict refs"